src/StringParser/Ini.php

```diff
@@ -0,0 +1,73 @@
+<?php
```

Comment:
This file is very similar to FileParser/Ini.php. We should share the similar functionalities in a common place. The same issue is present in Json, Xml, Yaml and in part in Php parsers.

Reply:
I've been thinking about this, but I don't know how exactly should I make this. Do you have some ideas?